tests/Functional/ConfigurationTest.php

```diff
@@ -95,22 +95,24 @@ public function testChangeComponentLibrary(): void {
       $this->drupalGet('<front>');
 
       // Assert that we load the EU component library.
-      $this->assertLinkContainsHref('/oe_theme/dist/eu/styles/ecl-eu-preset-website.css');
-      $this->assertLinkContainsHref('/oe_theme/dist/eu/styles/ecl-eu-preset-website-print.css');
+      $this->assertLinkContainsHref('/oe_theme/dist/preset-reset/styles/optional/ecl-reset.css');
+      $this->assertLinkContainsHref('/oe_theme/dist/eu/styles/ecl-eu.css');
+      $this->assertLinkContainsHref('/oe_theme/dist/eu/styles/ecl-eu-print.css');
       $this->assertLinkContainsHref('/oe_theme/css/style-eu.css');
 
-      $this->assertScriptContainsSrc('/oe_theme/dist/eu/scripts/ecl-eu-preset-website.js');
+      $this->assertScriptContainsSrc('https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js');
+      $this->assertScriptContainsSrc('/oe_theme/dist/eu/scripts/ecl-eu.js');
       $this->assertScriptContainsSrc('/oe_theme/js/ecl_auto_init.js');
 
       // Assert that the favicon provided by the theme is being used.
       $this->assertSession()->responseContains('/' . $active_theme . '/favicon.ico');
 
       // Assert that we do not load the EC component library.
-      $this->assertLinkNotContainsHref('/oe_theme/dist/ec/styles/ecl-ec-preset-website.css');
-      $this->assertLinkNotContainsHref('/oe_theme/dist/ec/styles/ecl-ec-preset-website-print.css');
+      $this->assertLinkNotContainsHref('/oe_theme/dist/ec/styles/ecl-ec.css');
+      $this->assertLinkNotContainsHref('/oe_theme/dist/ec/styles/ecl-ec-print.css');
       $this->assertLinkNotContainsHref('/oe_theme/css/style-ec.css');
 
-      $this->assertScriptNotContainsSrc('/oe_theme/dist/ec/scripts/ecl-ec-preset-website.js');
+      $this->assertScriptNotContainsSrc('/oe_theme/dist/ec/scripts/ecl-eu.js');
```

Comment:
This only passes because that path doesn't actually exist so it can't be found anyway.
```suggestion
      $this->assertScriptNotContainsSrc('/oe_theme/dist/ec/scripts/ecl-ec.js');
```